src/impl/User/model.py

```diff
@@ -29,7 +30,11 @@ class User(Base):
     terms_accepted: bool = Column(Boolean, default=True)
```

Comment:
move to the user config

Reply:
comentat el "terms_accepted: bool = Column(Boolean, default=True)"